Script explanations:

filters - Filter the short-read VCF files after variant calling with smoove
minimap - Map the long-read data to the CanFam3.1 reference genome
plcr - Create samplot/plotcritic files & folders
plcr_parse_reports - Summarize plotcritic results and create a folder for a 2nd pass
plcr_to_vcf - Isolate the variants that passed the curation and collect them in a vcf file
plink - Create PCAs for all the final variants (as well as a few reject files)
plink_50 - Create PCAs
plink_SNPs - Create PCAs for all the SNPs identified by Linn√©a Smeds
plink_subsets - Create PCAs for the Swedish & Finnish subsets
r_plotdepth - Create plots with coverage against number of identified variants
r_plotpca - Create PCA plots of all the generated plink files
samplot_dup_cmds - Generated by plcr.sh, contains code to generate the plotcritic folder
samplot_dup_cmds_pt2 - Generated by plcr.sh, contains code to generate the plotcritic folder
samplot_inv_cmds - Generated by plcr.sh, contains code to generate the plotcritic folder
setup - Pull programs & data required to run this project
smoove - Run smoove using a snakemake pipeline. Short-read structural variant calling
sniffles - Long-read structural variant calling
sniffles_pop - Population calling using sniffles. Long-read structural variant calling
trio_check - Generate files containing trios and analyze their genotypes
vcf_batchsplit - Split the dataset into two subsets: Swedish & Finnish subset